docs/reference/search/search-your-data/semantic-search-elser.asciidoc

```diff
@@ -183,42 +165,37 @@ GET _tasks/<task_id>
 ----
 // TEST[skip:TBD]
 
-You can also open the Trained Models UI, select the Pipelines tab under ELSER to
-follow the progress.
-
+You can also open the Trained Models UI, select the Pipelines tab under ELSER to follow the progress.
 
 [discrete]
 [[text-expansion-query]]
-==== Semantic search by using the `text_expansion` query
+==== Semantic search by using the `sparse_vector` query
 
-To perform semantic search, use the <<query-dsl-text-expansion-query, `text_expansion` query>>,
-and provide the query text and the ELSER model ID. The example below uses the
-query text "How to avoid muscle soreness after running?", the `content_embedding`
+To perform semantic search, use the <<query-dsl-sparse-vector-query, `sparse_vector` query>>, and provide the query text and the inference ID associated with your ELSER model.
+The example below uses the query text "How to avoid muscle soreness after running?", the `content_embedding`
 field contains the generated ELSER output:
 
 [source,console]
 ----
 GET my-index/_search
 {
    "query":{
-      "text_expansion":{
-         "content_embedding":{
-            "model_id":".elser_model_2",
-            "model_text":"How to avoid muscle soreness after running?"
-         }
+      "sparse_vector":{
+         "field": "content_embedding",
+         "inference_id": "my_elser_model",
```

Comment:
When referencing inference endpoints, prefer "endpoint" over "model" to highlight the difference between the two.

```suggestion
         "inference_id": "my-elser-endpoint",
```

Reply:
Understood, trying to fix these as we find them